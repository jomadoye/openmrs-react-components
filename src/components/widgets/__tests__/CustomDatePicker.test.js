import React from 'react';
import renderer from 'react-test-renderer';
import CustomDatePicker from '../CustomDatePicker';

let props;
let mountedComponent;

const getComponent = () => {
  if (!mountedComponent) {
    mountedComponent = renderer.create(<CustomDatePicker {...props} />);
  }
  return mountedComponent;
};

describe('Component: CustomDatePicker', () => {
  beforeEach(() => {
    props = {
    };
    mountedComponent = undefined;
  });

  it('renders properly', () => {
    const component = getComponent();
    expect(component.toJSON()).toMatchSnapshot();
  });
});
