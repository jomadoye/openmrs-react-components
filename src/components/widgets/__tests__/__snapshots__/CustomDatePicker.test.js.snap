// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: CustomDatePicker renders properly 1`] = `
<div>
  <div
    className="react-datepicker-wrapper"
  >
    <div
      className="react-datepicker__input-container"
    >
      <span>
        <span
          style={
            Object {
              "display": "flex",
              "justifyContent": "space-around",
            }
          }
        >
          <span
            className=""
          >
            
          </span>
          <input
            className="form-control"
            id={undefined}
            placeholder=""
            style={
              Object {
                "marginRight": "5px",
                "width": "100px",
              }
            }
            type="text"
            value="10/19/2018"
          />
        </span>
      </span>
    </div>
  </div>
</div>
`;
